plugins/weather/wunderground_

```diff
@@ -0,0 +1,356 @@
+#!/bin/sh -eu
```

Comment:
I would recommend to move the `-eu` argument to a separate `set -eu` call. At least I was quite confused, that your code seemed to rely on `-e`, but I failed to find an invocation of `set`.